features/unattached_status.feature

```diff
@@ -216,15 +450,69 @@ Feature: Unattached status
         """
 
         Examples: ubuntu release
-           | release | esm-apps | cc-eal | cis | cis-available | fips | esm-infra | ros | livepatch | usg | realtime-kernel |
-           | xenial  | yes      | yes    | cis | yes           | yes  | yes       | yes | yes       |     | no              |
-           | bionic  | yes      | yes    | cis | yes           | yes  | yes       | yes | yes       |     | no              |
-           | focal   | yes      | no     |     | yes           | yes  | yes       | no  | yes       | usg | no              |
-           | impish  | no       | no     | cis | no            | no   | no        | no  | no        |     | no              |
-           | jammy   | yes      | no     |     | no            | no   | yes       | no  | yes       | usg | yes             |
+           | release |
+           | focal   |
 
+    @series.jammy
+    @uses.config.machine_type.lxd.container
+    @uses.config.contract_token
+    Scenario Outline: Simulate status in a ubuntu machine
```

Comment:
I'm not convinced we need to test full simulate status output for each series. Isn't the availability logic the same as normal status?

Reply:
It should yes, but do you think should only test a subset of the series ?

Reply:
IMHO testing all series covers all use cases without us having to make any previous assumption. I think it is beneficial for the completeness of the integration tests.

Reply:
Okay! mine wasn't a strong opinion - let's keep as is